server/src/main/java/model/placement/fleet/FleetSerializer.java

```diff
@@ -0,0 +1,39 @@
+package model.placement.fleet;
+
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import model.placement.ship.PlacedShip;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+public class FleetSerializer {
+    private Fleet<PlacedShip> placedShipFleet;
+
+    private FleetSerializer(Fleet<PlacedShip> placedShipFleet) {
+        this.placedShipFleet = placedShipFleet;
+    }
+
+    public static FleetSerializer from(Fleet<PlacedShip> placedShipFleet) {
+        return new FleetSerializer(placedShipFleet);
+    }
+
+    public String serialize() {
+        Gson gson = new Gson();
+        Type type = new TypeToken<List<PlacedShip>>() {
+
```

Comment:
empty spaces, 
what are we living for? Abandon places?

https://www.youtube.com/watch?v=t99KH0TR-J4

Reply:
Done.